src/main/java/springcourse/phonebook/model/UserContact.java

```diff
@@ -0,0 +1,38 @@
+package springcourse.phonebook.model;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "contact")
+public class UserContact {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String contactName;
+    private String phoneNumber;
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public String getContactName() {
+        return contactName;
+    }
+
+    public void setContactName(String firstName) {
+        this.contactName = firstName;
+    }
+
+    public String getPhoneNumber() {
+        return phoneNumber;
+    }
+
+    public void setPhoneNumber(String lastName) {
+        this.phoneNumber = lastName;
+    }
```

Comment:
I highly encourage to use lombok: https://projectlombok.org/

Reply:
Fixed.
Lombok library is added.